### PR TITLE
fix (#minor); makerdao-ethereum; fixed procotoal side rev from wrong liquidation penality #1785

### PR DESCRIPTION
**Pull request by tnkrxyz**
This fixes the issue in protocol side revenue (#1785) due to liquidity penalty using incorrect decimals for the Cat V1 contract.

- cat v1: https://etherscan.io/address/0x78f2c2af65126834c51822f56be0d7469d7a523e#code
- cat v2: https://etherscan.io/address/0xa5679C04fc3d9d8b0AaB1F0ab83555b301cA70Ea#code

### deployment & validation
- deployment: https://thegraph.com/hosted-service/subgraph/tnkrxyz/makerdao-ethereum?version=pending
- validation dashboard: https://subgraphs.messari.io/subgraph?endpoint=https://api.thegraph.com/subgraphs/id/QmdN5XkS372qmKv1mDVq9SJZ68PsMb9YXrXRZS3Qha3Xfc&tab=protocol

### CODE REVIEW

**Comment by ishraq8** [comment]
awaiting sync

File: subgraphs/makerdao/protocols/makerdao/config/deployments/makerdao-ethereum/configurations.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "graftEnabled": true,
```
Comment (dmelotik) [inline subgraphs/makerdao/protocols/makerdao/config/deployments/makerdao-ethereum/configurations.json:2]: let's change this to false for the master branch

**Review by dmelotik — APPROVED** [review]
lgtm.

Also @ishraq8 it has passed the 2 spikes and it looks to be fixed, will let you confirm.

**Comment by ishraq8** [comment]
Confirmed, fixed the issue :)

also the recent increase in supply-side revenue is due to increase in dai savings rate

**Pull request merged**